app/javascript/lib/plot.js

```diff
@@ -295,11 +296,16 @@ PlotUtils.getLegendSortedLabels = function(countsByLabel) {
  * didn't have to do this.  See https://github.com/plotly/plotly.js/issues/5612
 */
 function countValues(array) {
-  return array.reduce((acc, curr) => {
-    acc[curr] ||= 0
-    acc[curr] += 1
-    return acc
-  }, {})
+  try {
+    return array.reduce((acc, curr) => {
+      acc[curr] ||= 0
```

Comment:
Ooh I don't think that I knew Javascript had this notation for conditional assignment!  I definitely appreciate this in Ruby.